3.4.10/Dockerfile

```diff
@@ -18,9 +19,9 @@ ENV ZOO_USER=zookeeper \
 
 # Add a user and make dirs
 RUN set -ex; \
-    adduser -D "$ZOO_USER"; \
-    mkdir -p "$ZOO_DATA_LOG_DIR" "$ZOO_DATA_DIR" "$ZOO_CONF_DIR"; \
-    chown "$ZOO_USER:$ZOO_USER" "$ZOO_DATA_LOG_DIR" "$ZOO_DATA_DIR" "$ZOO_CONF_DIR"
+    adduser -D -u 9001 "$ZOO_USER"; \
```

Comment:
It's hard to find examples of explicitly setting the UID in other official images. Is it really useful? Can you give some example images? I only found Node: https://github.com/nodejs/docker-node/blob/master/Dockerfile-alpine.template#L5-L6 It's more common to create both user and group, though. Maybe we should rather do this?